plugins/foreman_ansible/1.x/index.md

```diff
@@ -46,12 +46,16 @@ For Ansible versions older than 2.2:
 
 You can configure it via the following environment variables:
 
-* FOREMAN_URL: the URL of your Foreman installation (default "http://localhost:3000")
+* FOREMAN_URL: the URL of your Foreman installation (default "http://localhost:3000") - change it to your Foreman URL, e.g: "https://myforeman.mylocal.lan", it should not be port 3000.
 * FOREMAN_SSL_CERT: The public key when using SSL client certificates (default "/etc/foreman/client_cert.pem")
 * FOREMAN_SSL_KEY: The private key when using SSL client certificates (default  "/etc/foreman/client_key.pem")
 * FOREMAN_SSL_VERIFY: wether to verify SSL certificates. Use *False*
   to disable certificate checks. You can also set it to CA bundle (default is "True").
 
+If you're using Foreman without Katello, the SSL certificate and key are set at `/etc/httpd/conf.d/foreman.conf`. SSLCertificateFile, SSLCertificateKeyFile contain the values for FOREMAN_SSL_CERT and FOREMAN_SSL_KEY.
+
+If you are using the Katello plugin, the SSL certificate and key are set at `/etc/httpd/conf.d/05-foreman-ssl.conf`. SSLCertificateFile, SSLCertificateKeyFile contain the values for FOREMAN_SSL_CERT and FOREMAN_SSL_KEY. By default these are at `/etc/pki/katello/certs/katello-apache.crt` and `/etc/pki/katello/private/katello-apache.key`.
```

Comment:
I checked on 1.16 and the keys are at `/etc/httpd/conf.d/foreman.conf` and `/etc/httpd/conf.d/05-foreman.conf` respectively, let me know if you think that is wrong and should instead point to some other file

Reply:
It should *always* be ```05-foreman-ssl.conf```. Both with and without katello since puppet-foreman sets that up. In the katello scenario we just change the default answers from puppet locations to katello locations.

That said, I think in the katello scenario the correct certificate is the one we install for puppet server to talk back to foreman. It's guaranteed to be signed by the correct CA (remember there can be custom certificates signed by a diffferent CA). It's also the same function: report callbacks. See https://github.com/theforeman/puppet-certs#certificates-overview

Maybe @stbenjam has some good insight here, but I think it would make sense to implement this in the installer.